ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.22-periodics.yaml

```diff
@@ -0,0 +1,1068 @@
+periodics:
```

Comment:
Do we need it?

Reply:
We don't run them for other versions, so maybe not?

Reply:
I can remove it but not sure if the config jobs will pass.

Reply:
config does not pass if removed:
```
diff -Naupr .//ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.22-periodics.yaml /tmp/tmp.pUU7YGf2Rm/ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.22-periodics.yaml
--- .//ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.22-periodics.yaml	1970-01-01 00:00:00.000000000 +0000
+++ /tmp/tmp.pUU7YGf2Rm/ci-operator/jobs/openshift-knative/serverless-operator/openshift-knative-serverless-operator-release-1.22-periodics.yaml	2022-04-05 19:34:10.098611968 +0000
```

Reply:
We don't want the periodic jobs. We only want them for `main` but not for this release branch. They're defined in the ci-operator config (e.g. [here](https://github.com/openshift/release/pull/27571/files#diff-46b7739dde0304cfcae92fbd1cbcc525499630a00471be3b0fb39ce7be05d34dR167)) - all the checks with `cron` definition in them (including UI tests)

Reply:
So runnign again `make....` without periodics this part is not generated anymore.Makes sense.